### DISCUSSION TITLE
How to get the latest version?

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#2974] [Q&A] [answered]
`Check for updates` said:

```
Installed version: 3.9.0.3371
Available version: 3.9.0.3386
```

https://pascalabc.net/en/download shows:

`PascalABC.NET (version 3.9.0, build 3328, 30.07.2023, for Windows 7, 8, 10, Linux)`

So, where to get 3.9.0.3386?

**Comment by SunSerega** [comment, marked as answer]
https://pascalabc.net/downloads/pabcversion.txt

This is the true latest version of the currently listed installers. It's also what IDE checks. Download page lies.
